router/pkg/config/config.go

```diff
@@ -451,13 +451,18 @@ type PersistedOperationsCDNProvider struct {
 	URL string `yaml:"url,omitempty" envDefault:"https://cosmo-cdn.wundergraph.com"`
 }
 
-type ExecutionConfigStorageConfig struct {
+type ExecutionConfigStorage struct {
 	ProviderID string `yaml:"provider_id,omitempty" env:"EXECUTION_CONFIG_STORAGE_PROVIDER_ID"`
 	ObjectPath string `yaml:"object_path,omitempty" env:"EXECUTION_CONFIG_STORAGE_OBJECT_PATH"`
 }
 
+type ExecutionConfigFile struct {
+	Path string `yaml:"path,omitempty" env:"EXECUTION_CONFIG_FILE_PATH"`
```

Comment:
Is watch missing here?

Reply:
No, see comment. It is just a proposal how this interface allows for extensions.